### ISSUE TITLE
Complete build options for Pthread API

### ISSUE THREAD

**Issue by elfring** [infinisql/infinisql#13]
Would you like to add the configuration script "[AX_PTHREAD](http://www.gnu.org/software/autoconf-archive/ax_pthread.html)" to [your build specification](https://github.com/infinisql/infinisql/blob/f399773ce5f0cbff5dd11277dd26e6e1a2e93f26/configure.ac#L21)?

**Comment by mtravis15432** [comment]
Yes, it looks like this improves portability, with no drawbacks, so that's a good thing.
